src/validations/validations.ts

```diff
@@ -44,96 +73,112 @@ export function validateMetatile(metatile: number): void {
 }
 
 /**
- * Validates that the input `scaleSet` is valid
- * @param scaleSet the scale set to validate
- */
-export function validateScaleSet(scaleSet: ScaleSet): void {
-  const arr = [...scaleSet.scaleDenominators.entries()];
-  for (let i = 0; i < arr.length - 1; i++) {
-    if (arr[i][0] + 1 !== arr[i + 1][0]) {
-      throw new Error("scale set must have it's zoom levels ordered in ascending order and must be larger then the previous by 1");
-    }
-
-    if (arr[i][1] <= arr[i + 1][1]) {
-      throw new Error("scale set must have it's scales ordered in ascending order and must be larger then the previous");
-    }
-  }
-}
-
-/**
- * Validates that the input `tileGrid` is valid
- * @param tileGrid the tile grid to validate
+ * Validates that the input `tileMatrix` is valid
+ * @param tileMatrix the tile matrix to validate
  */
-export function validateTileGrid(tileGrid: TileGrid): void {
-  validateBoundingBox(tileGrid.boundingBox);
-  validateScaleSet(tileGrid.wellKnownScaleSet);
+export function validateTileMatrix(tileMatrix: TileMatrix): void {
+  // validateBoundingBox(tileMatrix.boundingBox);
+  // validateScaleSet(tileMatrix.wellKnownScaleSet);
```

Comment:
Why are these commented?

Reply:
removing

i'm planning on changing the validations